extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -23,6 +23,7 @@ amp-sticky-ad {
   z-index: 2;
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
+  margin-bottom: 0 !important;
```

Comment:
I see your point! Yes I should also add `padding-bottom: 0 !important`. But for `background` I don't know, I mean it should be customizable and I don't see why we shouldn't allow user to make sticky-ad background-color to the match the style of its parent.